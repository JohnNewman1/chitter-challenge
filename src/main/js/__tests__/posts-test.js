import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Posts from '../posts/posts';
import { configure } from 'enzyme';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('a test', function(){
    it('testing posts', function(){
      const tree = renderer.create(
        <Posts />
      ).toJSON();
      expect(tree).toMatchSnapshot();
    })
})