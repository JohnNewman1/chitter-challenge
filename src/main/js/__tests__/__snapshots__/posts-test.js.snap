// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a test testing posts 1`] = `
<article
  className="posts-main"
>
  <h1
    className="posts-title"
  >
    Posts
  </h1>
  <div
    className="posts-items"
  />
</article>
`;
